SwiftCompilerSources/Sources/Optimizer/InstructionSimplification/SimplifyAllocStack.swift

```diff
@@ -277,7 +277,7 @@ private extension AllocStackInst {
         // The instruction which defines the existential archetype must dominate the alloc_stack, because
         // after the transformation the alloc_stack will use the existential archetype.
         for openArchetypeOp in initExistential.typeDependentOperands {
```

Comment:
is openArchetypeOp still the right name for this?

Reply:
good point. I'll change it in a follow-up PR